README.md

```diff
@@ -1,24 +1,285 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <h1><b>Final Capstone Back-End</b></h1>
+</div>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📗 Table of Contents](#-table-of-contents)
+- [About](#about-project)
+  - [🛠 Built With](#-built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [ERD](#erd)
+    - [Front-End Project](#front-end)
+  - [💻 Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run linters](#run-linters)
+  - [👥 Authors](#authors)
+  - [🔭 Future Features](#future-features)
+  - [🤝 Contributing](#contributing)
+  - [⭐️ Show your support](#️show-your-support)
+  - [🙏 Acknowledgments](#acknowledgments)
+  - [📝 License](#license)
 
-* System dependencies
+---
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# About <a name="about-project"></a>
 
-* Database initialization
+**"Final Capstone Back-End"** is an API for saving records of different users, displaying different experiences, and also allowing the registered users to book or cancel an experience. It is made with Ruby on Rails and PostgreSQL.
 
-* How to run the test suite
+---
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- BUILT WITH -->
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails (v7.0.8)</a></li>
+    <li><a href="https://www.postgresql.org/docs/">PostgreSQL</a></li>
+    <li><a href="https://github.com/rswag/rswag">API Documentation (Swagger)</a></li>
+    <li><a href="https://hackernoon.com/how-to-write-your-first-tests-using-rspec-in-rails-applications-hhfk2bqs">Rspec</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Linters</summary>
+  <ul>
+    <li><a href="https://rubocop.org/">Rubocop</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<ul>
+  <li>Ruby on Rails framework</li>
+  <li>
+    Use Linters to check code quality
+    <ul>
+      <li>Rubocop</li>
+    </ul>
+  </li>
+  <li>MVC architectural/design pattern</li>
+  <li>Data persistent in PostgreSQL</li>
+  <li>RESTful API</li>
+  <li>Swagger to generate API doc</li>
+  <li>Users, Experiences and Reservations endpoints</li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- ERD -->
+
+### ERD  <a name="front-end"></a>
+
+On the following ERD you can see the structure of this project.
+
+<img src="assets/images/ERD.png" alt="ERD" width=900px />
+
```

Comment:
@AndreaM2429 I kindly suggest that you update the application's ER diagram so that the latest changes appear in the `experiences` table.

Reply:
Thanks, @NitBravoA92, you're completely right I forgot to update the diagram but now it's.